src/stdune/path.mli

```diff
@@ -63,7 +60,7 @@ module Build : sig
 
   val append_local : t -> Local.t -> t
 
-  val of_string : ?error_loc:Loc0.t -> string -> t
+  val parse_string_exn : ?error_loc:Loc0.t -> string -> t
```

Comment:
Maybe `error_loc` should be explicit here (`error_loc : Loc.t option`)? If you're parsing the user input, you'll want to specify Loc more often than not.

Reply:
Yh, actually maybe we should just require it, i.e.:

```suggestion
  val parse_string_exn : loc:Loc0.t -> string -> t
```

That will make it clearer that it's needed.

(we can also drop the `error_` prefix BTW, it is only named like this for historical reason)

Reply:
Yeah, good call.